agent/consul/grpc_services.go

```diff
@@ -0,0 +1,218 @@
+package consul
```

Comment:
Finding all the places we register grpc services on the was proving difficult. I have isolated all the service creations into this singular file and there is now a `setupGRPCServices` on the server to add those services to all the correct gRPC interfaces.